CHANGES.md

```diff
@@ -1,3 +1,11 @@
+1.2.0
+=====
+
+* [#159](https://github.com/serokell/universum/issues/159):
+  Remove `text-format` dependency.
+* [#164](https://github.com/serokell/universum/issues/164):
+  Don't reexport `log :: Floating a => a -> a`.
```

Comment:
I suggest that we write something like **Breaking change** for all breaking changes and provide a suggestion how to overcome it if it's not trivial. For example, removing `text-format` dependency is a breaking change which is not trivial to overcome, because `pretty` and `prettyL` don't exist in other librearies. A possible suggestion is to use the `fmt` function (I didn't use it, but I was told it can serve as a direct replacement).